dsn_test.go

```diff
@@ -62,18 +54,6 @@ func TestParseDSN(t *testing.T) {
 	if config.transactionIsolation != 8 {
 		t.Errorf("Expected transactionIsolation to be %d, got %d", 8, config.transactionIsolation)
 	}
-
```

Comment:
Keep the tests below to test authentication using the DSN.

Reply:
done